.github/workflows/e2e-tests.yaml

```diff
@@ -85,12 +86,26 @@ jobs:
             arch: "arm"
             family: "Ubuntu2204"
             kubernetes-version: "1.31.0"
+          # Since we only enable enforcing mode for SELinux in AL2023, it's easier to list it in "include"
+          # field rather than trying to exclude all other variants.
+          - cluster-type: "kops"
+            arch: "arm"
+            family: "AmazonLinux2023"
+            selinux-mode: "enforcing"
         exclude:
           - cluster-type: "kops"
             family: "Bottlerocket"
           - cluster-type: "eksctl"
             arch: "arm"
             family: "Bottlerocket"
+          - cluster-type: "eksctl"
+            family: "Ubuntu2004"
+          # We're mainly using AL2023 to test our CSI driver with enforced SELinux,
+          # AL2023 supports switching to enforcing mode via cloud-init, but,
+          # passing user-data (the way we configure cloud-init) is not supported on eksctl for AL2023.
+          # So, we're only running SELinux tests with kops.
```

Comment:
So I think we now figured out how to enforce SELinux via `eksctl` using:
```yaml
preBootstrapCommands:
    - "dnf install -y container-selinux"
    - "setenforce 1"
    - 'sed -i "s/SELINUX=permissive/SELINUX=enforcing/" /etc/selinux/config'
```

We should also enable SELinux on `eksctl` clusters.